### DISCUSSION TITLE
Logic Sound Library on Audiogridder server

### DISCUSSION THREAD

**Discussion by richiewinn** [apohl79/audiogridder#805] [Q&A]
I am thinking about buying a 2012 Mini Mac to act as an AudioGridder server hooked up via an ethernet cable to my 2010 iMac running Logic Pro X.  Is it simply a matter of relocating the Logic Sound Library onto the server and the server software will "find" it?  Thanks in advance and if this has been covered elsewhere apologies.  Richie

**Comment by JamesXXV** [comment]
you can enable network sharing in the 2012 Mac mini and just click on the network drive while installing the Logic Pro library, no audiogridder assistance required. Personally I wouldn't recommend using a 1 gigabit connection for samples, nvme ssd on the server side and a 10 gig connection might be okay. better and easier would be to upgrade the hdd to bigger ssd on the 2010 iMac.
https://support.apple.com/guide/mac-help/mh17131/mac

**Reply by richiewinn** [reply]
Thanks for the response.  If I understand correctly then I think what you suggest is I can install the Logic Pro Library on a network (using the MiniMac as a server perhaps?) without Audiogridder assistance?  That's fine but when I use a VST from the library the processing will be done on the box where Logic resides?  My aim with all of this is to reduce the pressure on the Logic box by spreading the load to another (the MiniMac) and use Audiogridder to provide the link back (via the ethernet) to the Logic box.

**Reply by Fittkenny** [reply]
audiogridder loads plugins, Logic sample library is not a plugin

**Reply by JamesXXV** [reply]
yes install the library on the networked drive, Audiogridder doesn't need to come in between, it can play sample libraries yes but only from plugins like Kontakt and it has no way of controlling logic's stock library. If you open the VST3 directly on the iMac then it will use local resources you need open an instance of the Audiogridder plugin and install power hungry VST's on the Mac mini and load them through Audiogridder.

**Comment by custa1200** [comment]
Would not be putting it on a network drive unless it’s connected over 10Gb Ethernet or thunderbolt.
